### PR TITLE
qa/tasks/cephfs/: make sure that the count/seek/size... are int type under py3

### PR DESCRIPTION
**Pull request by lxbsz**
The int_type / 2 will return a float value in py3, in py2 it will be int type.

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/tasks/cephfs/test_volumes.py [hunk 1]
```diff
@@ -2156,7 +2156,7 @@ def test_subvolume_snapshot_clone_fail_with_remove(self):
         self.fs.add_data_pool(new_pool)
 
         self.fs.mon_manager.raw_cluster_cmd("osd", "pool", "set-quota", new_pool,
-                                            "max_bytes", "{0}".format(pool_capacity / 4))
+                                            "max_bytes", "{0}".format(pool_capacity//4))
```
Comment (tchaikov) [inline qa/tasks/cephfs/test_volumes.py:2159]: please add spaces around `/`.

File: qa/tasks/cephfs/test_volumes.py [hunk 2]
```diff
@@ -496,7 +496,7 @@ def test_subvolume_resize_fail_quota_lt_used_size_no_shrink(self):
         self.assertEqual(usedsize, susedsize)
 
         # shrink the subvolume
-        nsize = usedsize/2
+        nsize = usedsize//2
```
Comment (tchaikov) [inline qa/tasks/cephfs/test_volumes.py:499]: could you add spaces around `/`?

File: qa/tasks/cephfs/test_volumes.py (hunk 1 above)
Comment (lxbsz) [reply qa/tasks/cephfs/test_volumes.py]: Yeah, done.

File: qa/tasks/cephfs/test_volumes.py (hunk 2 above)
Comment (lxbsz) [reply qa/tasks/cephfs/test_volumes.py]: Done.

**Comment by lxbsz** [comment]
fold this into https://github.com/ceph/ceph/pull/34726, closing this now.

**Pull request closed without merging**